src/native/LexFloatClient.h

```diff
@@ -268,6 +275,18 @@ LEXFLOATCLIENT_API int LF_CC DropFloatingLicense();
 */
 LEXFLOATCLIENT_API int LF_CC HasFloatingLicense();
 
+/*
+    FUNCTION: RequestOfflineFloatingLicense()
+
+    PURPOSE: Sends the request to lease the license from the LexFloatServer for offline usage.
+
+    RETURN CODES: LF_OK, LF_FAIL, LF_E_PRODUCT_ID, LF_E_LICENSE_EXISTS, LF_E_HOST_URL,
```

Comment:
@muneebkq Why is the parameter missing here?

Reply:
jsdoc modified